### DISCUSSION TITLE
Transfer logs from PowerShell and import into Kibana

### DISCUSSION THREAD

**Discussion by organcory** [Security-Onion-Solutions/securityonion#6494] [Unsupported Versions] [answered]
OK, not sure how to word this but here goes.  We are currently using Winlogbeats from host machines, but have an in  house powershell program that was created to analyze host data.  It is able to create its own event logs and export as .evtx.  We want to transfer those .evtx files straight into Kibana instead of the winlogbeats as it gives too much information that isnt needed.

**Comment by dougburks** [comment, marked as answer]
One option might be `so-import-evtx`:
https://docs.securityonion.net/en/2.3/so-import-evtx.html
